scikit-build-core.spec

```diff
@@ -0,0 +1,66 @@
+Name:           python-scikit-build-core
+Version:        0.2.2
+Release:        %{autorelease}
+Summary:        Build backend for CMake based projects
+
+License:        Apache-2.0
+URL:            https://github.com/scikit-build/scikit-build-core
+Source0:        https://github.com/scikit-build/scikit-build-core/archive/refs/tags/v%{version}.tar.gz
+
+BuildArch:      noarch
+BuildRequires:  python3-devel
+BuildRequires:  cmake
+BuildRequires:  ninja-build
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+Requires:       cmake
```

Comment:
What version(s) of Python does this target? This requires `packaging` also, and possibly a a few other things on older Python versions? Or is there some reason that it doesn't need that?

Reply:
So all of those are copied from `%pyproject_buildrequires`, except for the python version that is targeted. In that case it is purely which version is being released for that Fedora release. We just add/drop Fedora versions

Reply:
Ahh, okay.